basicsr/losses/losses.py

```diff
@@ -329,7 +329,7 @@ def get_target_label(self, input, target_is_real):
         target_val = (self.real_label_val if target_is_real else self.fake_label_val)
         return input.new_ones(input.size()) * target_val
 
-    def forward(self, input, target_is_real, is_disc=False):
+    def forward_single(self, input, target_is_real, is_disc=False):
```

Comment:
I think GANLoss itself is pretty complcated. 
I recommend to support multiscale discriminator loss in a new class that inherent the GANLoss.

Reply:
OK. I'll call `super().forward()` in the new loss class